torch/quantization/fx/quantization_patterns.py

```diff
@@ -288,10 +292,19 @@ def convert(self,
         if is_reference and self.binary_op in binary_reference_op_supported_dtypes and \
                 dtypes in binary_reference_op_supported_dtypes[self.binary_op]:
             if dtypes in binary_op_int8_dtypes:
-                args = load_arg(quantized=[torch.quint8, torch.qint8])(node.args)
-                args = load_arg(quantized=torch.float)(node.args)
-                kwargs = load_arg(quantized=torch.float)(node.kwargs)
-                op_out = quantized_graph.node_copy(node, load_arg(quantized=torch.float))
+                args = load_arg(quantized=[torch.quint8, torch.qint8])(self.binary_op_node.args)
+                args = load_arg(quantized=torch.float)(self.binary_op_node.args)
+                kwargs = load_arg(quantized=torch.float)(self.binary_op_node.kwargs)
+                op_out = quantized_graph.node_copy(self.binary_op_node, load_arg(quantized=torch.float))
+
+                def modified_load_arg(n: Node):
```

Comment:
is there a way to avoid this?  Can we make the original definition of `load_arg` handle this instead?

Reply:
this is a hack that removes the `env` argument, I think we don't want to spend too much time on this, I think this will be gone after we do a refactor that removes `env` for convert

Reply:
makes sense, maybe we could add a comment to the original `load_arg` mentioning that some callsites may override it?

Reply:
OK will add this in a later PR, landing this now